storage/txcache/txCache.go

```diff
@@ -61,13 +64,21 @@ func (cache *TxCache) AddTx(tx *WrappedTransaction) (ok bool, added bool) {
 		cache.doEviction()
 	}
 
-	ok = true
-	added = cache.txByHash.addTx(tx)
-	if added {
-		cache.txListBySender.addTx(tx)
-		cache.monitorTxAddition()
+	addedInByHash := cache.txByHash.addTx(tx)
+	addedInBySender, evicted := cache.txListBySender.addTx(tx)
+	if addedInByHash != addedInBySender {
+		log.Trace("TxCache.AddTx(): slight inconsistency detected:", "name", cache.name, "tx", tx.TxHash, "sender", tx.Tx.GetSndAddr(), "addedInByHash", addedInByHash, "addedInBySender", addedInBySender)
```

Comment:
This can happen  when two go-routines concur to add the same transaction:
 - A adds to "txByHash"
 - B won't add to "txByHash" (duplicate)
 - B adds to "txListBySender"
 - A won't add to "txListBySender" (duplicate)

I've added a comment in the code now.